### ISSUE TITLE
Legion Toolkit is no longer detecting cpu utilization and core clock

### ISSUE THREAD

**Issue by aeensf** [BartoszCichecki/LenovoLegionToolkit#1566]
### Version

2.25.1

### OS

Windows 10 pro 22H2 build 19045

### Device

Legion 5 16IRX9

### BIOS version

NMCN28WW

### What is wrong?

After switching the GPU working mode from dGPU to Hybrid and restarting, I selected Discrete GPU in the BIOS. Since then, the Legion Toolkit no longer detects CPU utilization or core clock. 
(In the task manager cpu utilization is still detectable)

### What did you try already?

switching between different gpu modes, as well as Hybrid-Auto and Hybrid iGPU.
switching between Discrete GPU and Dynamic GPU in the BIOS.
reinstalling lenovo legion toolkit

### How to reproduce the bug?

happened after switching the GPU working mode from dGPU to Hybrid and restarting and then selecting Discrete GPU in the BIOS.

### What is the behavior that you expected?

Expted cpu utilization to be shown.

### Logs

[log_2025_02_04_12_26_33.txt](https://github.com/user-attachments/files/18657050/log_2025_02_04_12_26_33.txt)

**Comment by BartoszCichecki** [comment]
Your performance counters are broken:

```
System.InvalidOperationException: Cannot load Counter Name data because an invalid index '' was read from the registry. Performance counters on the machine may need to be repaired.
   at Hashtable System.Diagnostics.PerformanceCounterLib.GetStringTable(bool isHelp)
   at Hashtable System.Diagnostics.PerformanceCounterLib.get_NameTable()
   at Hashtable System.Diagnostics.PerformanceCounterLib.get_CategoryTable()
   at bool System.Diagnostics.PerformanceCounterLib.CounterExists(string category, string counter, ref bool categoryExists) x 2
   at void System.Diagnostics.PerformanceCounter.InitializeImpl()
   at new System.Diagnostics.PerformanceCounter(string categoryName, string counterName, string instanceName, bool readOnly) x 2
   at void LenovoLegionToolkit.Lib.Utils.SafePerformanceCounter.TryCreateIfNeeded()
```

**Comment by aeensf** [comment]
um.... okay... but if they're broken how can the taskmanager detect cpu utilization?...

**Comment by BartoszCichecki** [comment]
I don't know, ask Microsoft.
